### ISSUE TITLE
Exception occurs when trying to view  local timeline

### ISSUE THREAD

**Issue by aadilayub** [ihabunek/toot#175]
![image](https://user-images.githubusercontent.com/31581044/85671690-8f825d80-b6db-11ea-9a5b-a22de4bd4841.png)

I get an exception every time I try to view my instance's local timeline. This occurs both in the tui and when using the `toot timeline` command. At first I thought it was because I used the login_cli option to authenticate, but I reauthenticated and am still getting the error.

Ran `toot auth` to confirm that I'm authenticated properly as well:

![image](https://user-images.githubusercontent.com/31581044/85672083-f56ee500-b6db-11ea-8e1d-5be4386702c8.png)

**Comment by ihabunek** [comment]
@aadilayub Could you scroll down to the bottom of the exception and tell me what it says?

Alternatively you can run tui with:

```

**Comment by ihabunek** [comment]
Oops, closed accidentally.

**Comment by aadilayub** [comment]
here you go:

```
DEBUG:toot:<<< <Response [401]>
DEBUG:toot:<<< b'{"error":"This method requires an authenticated user"}'
ERROR:toot.tui.app:This method requires an authenticated user
Traceback (most recent call last):
  File "/usr/local/Cellar/toot/0.26.0/libexec/lib/python3.8/site-packages/toot/tui/app.py", line 158, in _done
    result = future.result()
  File "/usr/local/opt/python@3.8/Frameworks/Python.framework/Versions/3.8/lib/python3.8/concurrent/futures/_base.py", line 432, in result
    return self.__get_result()
  File "/usr/local/opt/python@3.8/Frameworks/Python.framework/Versions/3.8/lib/python3.8/concurrent/futures/_base.py", line 388, in __get_result
    raise self._exception
  File "/usr/local/opt/python@3.8/Frameworks/Python.framework/Versions/3.8/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/usr/local/Cellar/toot/0.26.0/libexec/lib/python3.8/site-packages/toot/tui/app.py", line 257, in _load_statuses
    data = next(self.timeline_generator)
  File "/usr/local/Cellar/toot/0.26.0/libexec/lib/python3.8/site-packages/toot/api.py", line 179, in _anon_timeline_generator
    response = http.anon_get(url, params)
  File "/usr/local/Cellar/toot/0.26.0/libexec/lib/python3.8/site-packages/toot/http.py", line 64, in anon_get
    return process_response(response)
  File "/usr/local/Cellar/toot/0.26.0/libexec/lib/python3.8/site-packages/toot/http.py", line 45, in process_response
    raise ApiError(error)
toot.exceptions.ApiError: This method requires an authenticated user
```

**Comment by ihabunek** [comment]
This should be fixed in `0.27.0`, could you please upgrade and check?

**Comment by aadilayub** [comment]
@ihabunek checked, it works!
